server/public/css/nodeViewer.css

```
@@ -156,3 +155,10 @@ pre *::selection {
 .run:active {
   background-color: rgb(0, 235, 0);
 }
+
+.readme-scrollable {
+  overflow-y: auto;
+  overflow-x: hidden;
+  height: calc(100vh - 45px);
+  z-index: -100;
+}
```

Ada minor error dari [codacy](https://app.codacy.com/gh/bellshade/Javascript/pullRequest?prid=9232556) yang mengharuskan ada new line di akhir file, dan juga ini dipake disebelah mana ya css nya ?

```suggestion
}

```